src/jit/lsra.cpp

```diff
@@ -1122,6 +664,7 @@ LinearScan::LinearScan(Compiler* theCompiler)
 #endif // MEASURE_MEM_ALLOC
     , intervals(LinearScanMemoryAllocatorInterval(theCompiler))
     , refPositions(LinearScanMemoryAllocatorRefPosition(theCompiler))
+    , listNodePool(theCompiler, 8)
```

Comment:
It is possible to replace the 8 with an enum defined constant?

Reply:
Note that this was pre-existing, but was in a different place. I've changed it to be a default on the class.